test/generate-manifest.test.js

```diff
@@ -20,14 +21,20 @@ describe('Generate Complete Manifest', function () {
 
   before(function () {
     // Create stub for generateNonce function
-    const stub = sinon.stub(cryptoUtils, 'generateNonce')
-    stub.returns('123450325')
+    const generateNonce = sinon.stub(cryptoUtils, 'generateNonce')
```

Comment:
It's probably safer to move this into the `beforeEach` so that you can use the sandbox to stub the method. Otherwise it could stay stubbed in other test files where you don't want it to be stubbed